ui/de-lib/src/main/java/org/iplantc/de/client/util/OntologyUtil.java

```diff
@@ -69,7 +71,16 @@ public Avu convertHierarchyToAvu(OntologyHierarchy hierarchy) {
         Avu avu = avuFactory.getAvu().as();
         avu.setAttr(getAttr(hierarchy));
         avu.setValue(hierarchy.getIri());
-        avu.setUnit(hierarchy.getLabel());
+        avu.setUnit("");
```

Comment:
That should be correct: the `Unit` should be required, even if it's an empty string.